satpy/readers/__init__.py

```diff
@@ -611,4 +611,9 @@ def load_readers(filenames=None, reader=None, reader_kwargs=None,
                 remaining_filenames)))
     if not reader_instances:
         raise ValueError("No supported files found")
+    elif not all([list(r.available_dataset_ids)
+                  for r in reader_instances.values()]):
```

Comment:
Is there any reason or use case that this should maybe be `not any(...)`? That is, we could allow one or more readers to not have any datasets available but not *all* of them. Thinking if a user has multiple readers available and wants to load some dataset from at least one of them.

Granted the multiple reader functionality isn't really used by anyone right now.

Reply:
:+1: Good idea!